datahub-web-react/build.gradle

```diff
@@ -79,7 +79,7 @@ task yarnServe(type: YarnTask, dependsOn: [yarnInstall, yarnGenerate]) {
 
 task yarnTest(type: YarnTask, dependsOn: [yarnInstall, yarnGenerate]) {
   // Explicitly runs in non-watch mode.
-  args = ['run', 'test', 'run']
+  args = ['run', project.hasProperty('withCoverage') ? 'test-coverage' : 'test', 'run']
```

Comment:
Where is this property being set? We would want the coverage to be generated when gradle runs the `test` task (like all other projects).

Reply:
Good catch, should be added to the frontend build in the build-and-test workflow but in general doesn't need to be run in other places I think. 
`name: Gradle build (and test) for frontend`